Content.Client/Xenoarchaeology/Ui/NodeScannerDisplay.xaml.cs

```diff
@@ -0,0 +1,54 @@
+using Content.Client.UserInterface.Controls;
+using Content.Shared.Xenoarchaeology.Equipment.Components;
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+
+namespace Content.Client.Xenoarchaeology.Ui;
+
+[GenerateTypedNameReferences]
+public sealed partial class NodeScannerDisplay : FancyWindow
+{
+    [Dependency] private readonly IEntityManager _ent = default!;
+
+    public NodeScannerDisplay(EntityUid scannerEntityUid)
+    {
+        RobustXamlLoader.Load(this);
+
+        IoCManager.InjectDependencies(this);
+
+        var scannerComponent = _ent.GetComponent<NodeScannerComponent>(scannerEntityUid);
+
+        Update((scannerEntityUid, scannerComponent));
+    }
+
+    public void Update(Entity<NodeScannerComponent> ent)
+    {
+        var triggeredNodesSnapshot = ent.Comp.TriggeredNodesSnapshot;
+        if (triggeredNodesSnapshot.Count == 0)
+        {
+            NoActiveNodeDataLabel.Visible = true;
+            ActiveNodesList.Visible = false;
+            ActiveNodesList.Children.Clear();
+        }
+        else
+        {
+            NoActiveNodeDataLabel.Visible = false;
+            ActiveNodesList.Visible = true;
+
+            ActiveNodesList.Children.Clear();
+
+            foreach (var nodeId in triggeredNodesSnapshot)
+            {
+                var nodeLabel = new Button
+                {
+                    Text = nodeId,
+                    Margin = new Thickness(15, 10, 0, 0),
+                    MaxHeight = 40,
+                    Disabled = true
```

Comment:
on the first poin - 
On this thing u are whole worldly wrong my dear. Its actually giving the only info we are missing - WHAT ACTUALLY GOD ACTIVATED - when we are unlocking stuff. No other system is displaying this normally, only hints. And as u need ONLY required nodes to trigger for unlock - this gives EXACTLY needed tool to understand WHAT THE HELL is going wrong. This will give some self-discoverability to players imo. Otherwise - they will still going to sit and not understand why the fuk new node is not unlocking (while problem is - they trigger more nodes that they actually need).
I tried to set this to list of activated nodes and then i got into situation when i am having no idea what i happening and why artifact is triggered 90% of the time by itself!

On second point - 
yea, but actually no. Second ui would make small hand-held device feel too complex imo. Maybe i'll do it but not in this PR, it feels like current feature is more important for people to be able to WORK with artifact on basic level (unlock stuff). More core. 

On third idea - 
i'm not sure we should have them on client, so i'd better not. Maybe after i'll put some more noita-like stuff it'd be very important to not network them? I'm just not sure >_<